x/distribution/types/dec_coin.go

```diff
@@ -176,3 +176,13 @@ func (coins DecCoins) AmountOf(denom string) sdk.Dec {
 		}
 	}
 }
+
+// returns the amount of a denom from deccoins
```

Comment:
Just curious, why don't we have `DecCoins` in `types/`?

Reply:
another PR...